tap_shopify/streams/compatibility/product_compatibility.py

```diff
@@ -103,8 +112,8 @@ def _convert_variants(self):
                     "taxable": variant["taxable"],
                     "title": variant["title"],
                     "updated_at": variant["updatedAt"],
-                    "weight": variant["weight"],
-                    "weight_unit": variant["weightUnit"],
+                    "weight": variant["inventoryItem"]["measurement"]["weight"]["value"],
+                    "weight_unit": variant["inventoryItem"]["measurement"]["weight"]["unit"],
```

Comment:
Is this safe to unwrap like this? I'm worried we'll get a `KeyError` if `weight` is `None`

Reply:
@hsyyid Good point, I'll use a `.get` here. I'm not sure if weight will ever be `None` but I imagine it could be.